lib/paths.cf

```diff
@@ -570,6 +570,27 @@ bundle common paths
 
       "path[shadow]"       string => "/etc/security/passwd";
 
+    termux::
+      # some adjustments for the basics
+      "path[tar]"           string => "/usr/bin/tar";
+      "path[true]"          string => "/usr/bin/true";
+      "path[false]"         string => "/usr/bin/false";
+      "path[sysctl]"        string => "/usr/bin/sysctl";
+
+      # now, mangle the values by prepending the TERMUX_PREFIX
+      "root_path" string => "/data/data/com.termux/files";
+      # a prefix to use elsewhere
+      "prefix_path" string => "$(root_path)$(const.dirsep)usr";
+      "tmp_paths" slist => getindices("path");
+      "tmp_path[$(tmp_paths)]" string => "$(root_path)$(path[$(tmp_paths)])";
+      "path[$(tmp_paths)]" string => "$(tmp_path[$(tmp_paths)])";
+
+    !(termux|windows)::
+      # reasonable defaults for unix systems to allow for writing
+      # more portable paths between termux and other systems
+      "root_path" string => "/";
+      "prefix_path" string => "/usr";
```

Comment:
Where is prefix_path used?

Reply:
intended to be shared, so needs docs I would say.

Reply:
I will find a better name and add docs+example like

```
$(thing)/etc/sshd/sshd_config
```

So that `$(thing)` will be "right" on both unix systems and termux.